client/helpers.go

```diff
@@ -25,6 +25,19 @@ func IgnoreAccessDeniedServiceDisabled(err error) bool {
 	return false
 }
 
+func IgnoreWithInvalidAction(err error) bool {
+	if IgnoreAccessDeniedServiceDisabled(err) {
+		return true
+	}
+	var ae smithy.APIError
+	if errors.As(err, &ae) {
+		if ae.ErrorCode() == "InvalidAction" {
```

Comment:
Why we need to ignore InvalidAction? Looks like something is wrong with our API or how we call it?

Reply:
It only occurs on specific regions eu-west-3 etc' (only 4 regions failed) I doubled checked with CLI returns same result